### DISCUSSION TITLE
How can I add navigation arrows to a coverflow slider?

### DISCUSSION THREAD

**Discussion by alivault** [nolimits4web/swiper#4045] [Q&A] [answered]
I have a slider [here](https://aliabbas.webflow.io/brands) that has a coverflow effect. I have the swiper-button-prev and next elements in there plus  navigation: {
    nextEl: '.swiper-button-next',
    prevEl: '.swiper-button-prev',
  },
in the script. What am I doing wrong?

**Comment by vltansky** [comment, marked as answer]
You are not importing navigation styles. Everything fine with JS and config.
You probably importing only core styles `swiper.scss`.

> SCSS Styles
> SCSS styles are also separate styles for core version and components:
> 
> swiper.scss - only core Swiper styles
> components/a11y/a11y.scss - styles required for A11y component
> components/controller/controller.scss - styles required for A11y component
> components/effect-coverflow/effect-coverflow.scss - styles required for Coveflow Effect component
> components/effect-cube/effect-cube.scss - styles required for Cube Effect component
> components/effect-fade/effect-fade.scss - styles required for Fade Effect component
> components/effect-flip/effect-flip.scss - styles required for Flip Effect component
> components/lazy/lazy.scss - styles required for Lazy component
> components/navigation/navigation.scss - styles required for Navigation component
> components/pagination/pagination.scss - styles required for Pagination component
> components/scrollbar/scrollbar.scss - styles required for Scrollbar component
> components/thumbs/thumbs.scss - styles required for Thumbs component
> components/zoom/zoom.scss - styles required for Zoom component

https://swiperjs.com/api/ see styles section
